packages/base/any/kernels/4.19-lts/configs/x86_64-all/x86_64-all.config

```diff
@@ -3311,6 +3311,40 @@ CONFIG_USB_STORAGE=y
 # CONFIG_USB_GPIO_VBUS is not set
 # CONFIG_USB_ISP1301 is not set
 # CONFIG_USB_GADGET is not set
+#
+# USB Peripheral Controller
```

Comment:
If I understand the ONL build system correctly, you change the Linux configuration for all devices here. It’d be great if you put that in a separate commit, and maybe even do one commit for each feature you enable in the Linux kernel with the motivation/explanation why you need that configuration. That way, people looking through `git log` know what is going on, and it’s also easier to revert.

Reply:
Yes, sure. Because there is cp2130 in our S9500-22XST system, we would need to enable USB ACM driver for this.
I will make this as separate commit.